File: kala-game-service/src/main/java/org/jesperancinha/games/kalagameservice/KalaGameServiceApplication.java
package org.jesperancinha.games.kalagameservice;

import org.jesperancinha.games.kalagameservice.model.Stone;
import org.jesperancinha.games.kalagameservice.repository.KalaStoneRepository;
import org.springframework.boot.ApplicationArguments;
import org.springframework.boot.ApplicationRunner;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity;

@SpringBootApplication
@EnableWebSecurity
public class KalaGameServiceApplication implements ApplicationRunner {

    private final KalaStoneRepository stoneRepository;

    public KalaGameServiceApplication(KalaStoneRepository stoneRepository) {
        this.stoneRepository = stoneRepository;
    }

    public static void main(String[] args) {
        SpringApplication.run(KalaGameServiceApplication.class, args);
    }

    @Override
    public void run(ApplicationArguments args) throws Exception {
        this.stoneRepository.save(Stone.builder().build());
    }
}

File: kala-game-service/src/main/kotlin/org/jesperancinha/games/kalagameservice/KalaGameServiceApplication.kt
package org.jesperancinha.games.kalagameservice

import org.jesperancinha.games.kalagameservice.KalaGameServiceApplication
import org.springframework.boot.ApplicationArguments
import org.springframework.boot.ApplicationRunner
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.data.jpa.repository.config.EnableJpaRepositories
import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity

@SpringBootApplication
@EnableWebSecurity
@EnableJpaRepositories
open class KalaGameServiceApplication : ApplicationRunner {
    @Throws(Exception::class)
    override fun run(args: ApplicationArguments) {
    }

    companion object {
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplication.run(KalaGameServiceApplication::class.java, *args)
        }
    }
}